src/chainer.js

```diff
@@ -12,7 +13,7 @@ function chainer(functions) {
     }
```

Comment:
This code doesn't seem to fulfill the task. There is no chaining of unary functions. The returned function must take an input and apply the given unary functions in the order they were given, returning the final result.